client/management/v3/zz_generated_kontainer_driver.go

```diff
@@ -0,0 +1,137 @@
+package client
+
+import (
+	"github.com/rancher/norman/types"
+)
+
+const (
+	KontainerDriverType                      = "kontainerDriver"
+	KontainerDriverFieldActive               = "active"
+	KontainerDriverFieldActualURL            = "actualUrl"
+	KontainerDriverFieldAnnotations          = "annotations"
+	KontainerDriverFieldBuiltIn              = "builtIn"
+	KontainerDriverFieldChecksum             = "checksum"
+	KontainerDriverFieldConditions           = "conditions"
+	KontainerDriverFieldCreated              = "created"
+	KontainerDriverFieldCreatorID            = "creatorId"
+	KontainerDriverFieldExecutablePath       = "executablePath"
+	KontainerDriverFieldLabels               = "labels"
+	KontainerDriverFieldName                 = "name"
+	KontainerDriverFieldOwnerReferences      = "ownerReferences"
+	KontainerDriverFieldRemoved              = "removed"
+	KontainerDriverFieldState                = "state"
+	KontainerDriverFieldTransitioning        = "transitioning"
+	KontainerDriverFieldTransitioningMessage = "transitioningMessage"
+	KontainerDriverFieldUIURL                = "uiUrl"
+	KontainerDriverFieldURL                  = "url"
+	KontainerDriverFieldUUID                 = "uuid"
+	KontainerDriverFieldWhitelistDomains     = "whitelistDomains"
+)
+
+type KontainerDriver struct {
+	types.Resource
+	Active               bool              `json:"active,omitempty" yaml:"active,omitempty"`
+	ActualURL            string            `json:"actualUrl,omitempty" yaml:"actualUrl,omitempty"`
+	Annotations          map[string]string `json:"annotations,omitempty" yaml:"annotations,omitempty"`
+	BuiltIn              bool              `json:"builtIn,omitempty" yaml:"builtIn,omitempty"`
```

Comment:
make either readonly or drop it completely

Reply:
Marked as `noupdate`.
Added logic to formatter.